pkg/param/pagination/pagination.go

```diff
@@ -1,33 +1,32 @@
 package pagination
 
-// GetPage - pagination
-type GetPage struct {
+// Page - This is used for batch processing. User may want to process
+// limited objects instead of processing the whole object list . Specially when there are
+// 100s of objects exists on Array e.g. volumes, snapshots etc.
+type Page struct {
 	StartRow  *int
 	EndRow    *int
 	PageSize  *int
 	TotalRows *int
 }
 
 // ContainsMore - returns true if there are more pending objects then page size.
-func (page *GetPage) ContainsMore() bool {
+func (page *Page) ContainsMore() bool {
```

Comment:
With the flow I suggested in an earlier comment, I'm not sure we gain much by overloading the Page structure for use in both input and output.  We could use separate structures, with the TotalRows attribute and this function as part of just the output structure.  It would align better with the model of the underlying Nimble REST API.  I'm not sure either way is significantly better than the other, but it seems worth considering.